chains/evm/contracts/offRamp/CCVAggregator.sol

```diff
@@ -191,11 +188,15 @@ contract CCVAggregator is ITypeAndVersion, Ownable2StepMsgSender {
     if (ccvs.length != ccvData.length) {
       revert InvalidCCVDataLength(ccvs.length, ccvData.length);
     }
+    if (message.receiver.length != 20) {
+      revert Internal.InvalidEVMAddress(message.receiver);
```

Comment:
I've seen this same error thrown when the address is required to be padded. Not a big deal, just noting it.